src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2268,10 +2100,10 @@ void
 MavlinkReceiver::handle_message_collision(mavlink_message_t *msg)
 {
 	mavlink_collision_t collision;
-	collision_report_s t = {};
-
 	mavlink_msg_collision_decode(msg, &collision);
 
+	collision_report_s t{};
```

Comment:
This is nitpicky and not your doing, but because single letter variables are dangerous to begin with, and because `t` was already used above this for `transponder_report_t t{}`, this might be a good time to change this single letter variable to `cr` or something more descriptive.

Reply:
Updated.